internal/kubectl/kubectl.go

```diff
@@ -131,11 +133,11 @@ func IsPodReady(namespace string, labelName string, labelValue string) (bool, er
 	return true, nil
 }
 
-//CheckKubectlVersion assures that the kubectl version used is compatible
-func CheckKubectlVersion() error {
```

Comment:
How about using `semver` here? Like I did for minikube: https://github.com/kyma-incubator/kyma-cli/blob/master/internal/minikube/minikube.go#L48

Reply:
I was too lazy to change it, will do it

Reply:
I did it